frontend/packages/network-attachment-definition-plugin/src/models/index.ts

```diff
@@ -22,6 +23,7 @@ export const SriovNetworkNodePolicyModel: K8sKind = {
   abbr: 'SRNNPM', // TODO check on this
```

Comment:
These abbreviations are too long. We'll want something shorter. Max is 4 characters, but we really try to keep to 3 when possible.

Definitely don't include the `M` since "model" is an internal console concept :)